linode/placementgroup/framework_models_unit_test.go

```diff
@@ -30,7 +30,7 @@ func TestFlattenPGModel(t *testing.T) {
 		},
 	}
 
-	model := PlacementGroupModel{}
+	model := PlacementGroupResourceModel{}
```

Comment:
I noticed this fell out of parity with the introduction of the singular data source so I went ahead and fixed it here :+1: